pkg/controller/deployment/OWNERS

```diff
@@ -2,10 +2,8 @@ approvers:
 - janetkuo
 - nikhiljindal
 - kargakis
-- mfojtik
```

Comment:
I agree @mfojtik wasn't much active lately, the secrecy of how this was done is bothering :/ 

Would be nice to at least tag the person in question and @kubernetes/sig-apps-pr-reviews

Reply:
> I agree @mfojtik wasn't much active lately, the secrecy of how this was done is bothering :/
> 
> Would be nice to at least tag the person in question and @kubernetes/sig-apps-pr-reviews

The secrecy (no tagging), alacrity (one hour), and unilateral (one company) nature of the pull/review/merge process here is out of keeping with our usual processes.

Reply:
Agreed and I apologize. That was definitely not the intent.

Better late than never, @mfojtik can you review this PR now and let us know if the change looks good to you?
Also @mikedanese for pkg/controller/daemon/OWNERS.

@janetkuo Can you please also update the PR description to highlight these changes.

Reply:
@nikhiljindal I work closely with @tnozicka and ability to review/approve his fixes improves his productivity. If possible I would like to stay as approver there. However I understand that I'm not super-active last couple months (because of other work).

Reply:
Apologize for not making it clear in the first place and for the troubles it caused. @mfojtik's approver ownership was first approved by me in #40535. I missed that @mfojtik was added as an approver instead of a reviewer while approving that PR, and only noticed it recently. I still want to keep @mfojtik as a reviewer but I do expect more deployment work from deployment approver: https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Amfojtik+is%3Aclosed+deployment 

I hope you'll understand that this is done with no bad intentions. I'll open a PR to add @mfojtik as deployment approver and continue the discussions there.